index.js

```diff
@@ -1141,3 +1144,38 @@ function setHeaders (res, headers) {
     res.setHeader(key, headers[key])
   }
 }
+
+/**
+ * Get size of a given file (used especially for /proc/* files)
+ *
+ * @param {string} path
+ * @private
+ */
+function getSize (path) {
+	const fd = fs.openSync(path, 'r');
+	const tmpBufMinLen = 4096 * 2;
+	const tmpBufMaxLen = 4096 * 8;
+	let tmpBuf = Buffer.allocUnsafeSlow(tmpBufMinLen);
+	let pos = 0;
+	let bytesRead;
+	let buf = tmpBuf;
+	let length = buf.length;
+	do {
+		bytesRead = fs.readSync(fd, buf, pos, buf.length - pos, null);
```

Comment:
Performing sync operations will halt processing of parallel requests of the http server. You will need to use the async version of the functions, not the sync versions.